runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -221,6 +245,70 @@ else if (event.getGroupId() == WidgetID.BARROWS_PUZZLE_GROUP_ID)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		if (isInCrypt())
```

Comment:
Instead of nesting all of this code in an `if` block, change this to a guard clause:

```java
if (!isInCrypt())
{
	return;
}

// rest of code goes here
```